idl/github.com/uber/cadence/shared.thrift

```diff
@@ -54,6 +54,24 @@ exception QueryFailedError {
   1: required string message
 }
 
+enum WorkflowIdReusePolicy {
+  /*
+   * allow start a workflow execution using the same workflow ID,
+   * when workflow not running, and the last execution close state is in
+   * [closed, cancelled, timeouted, failed].
```

Comment:
i think closed is normal completed, do you allow in this case?

Reply:
this is actually a typo, should be terminated